.env

```diff
@@ -71,13 +71,14 @@ NEXT_PUBLIC_SITE_DOMAIN=main
 NEXT_PUBLIC_SITE_LANGUAGES=en
 NEXT_PUBLIC_SITE_DEFAULT_LANGUAGE=en
 
-# apm, only used when API_ENABLE_APM
-KIBANA_PORT=5601
-APM_URL=http://localhost:8200
-
 # elastic
 ELASTIC_HOST=localhost
 ELASTIC_PORT=9200
 ELASTIC_USERNAME=
 ELASTIC_PASSWORD=
 ELASTIC_FULL_TEXT_SEARCH_INDEX=full-text-search
```

Comment:
not directly related to the PR, but these values are unused

Reply:
#990